tsl/src/partialize_agg.c

```diff
@@ -52,14 +49,12 @@ is_vectorizable_agg_path(PlannerInfo *root, AggPath *agg_path, Path *path)
 	DecompressChunkPath *decompress_path = (DecompressChunkPath *) path;
 	Assert(decompress_path->custom_path.custom_paths != NIL);
 
-	Path *compressed_path = linitial(decompress_path->custom_path.custom_paths);
-
 	/* Hypertable compression info is already fetched from the catalog */
 	Assert(decompress_path->info != NULL);
 	Assert(decompress_path->info->hypertable_compression_info != NULL);
 
 	/* No filters are supported at the moment */
-	if (is_restricted_path(path) || is_restricted_path(compressed_path))
+	if ((list_length(path->parent->baserestrictinfo) > 0 || path->parent->joininfo != NULL))
```

Comment:
Shouldn't the comment be a bit reworded?

Reply:
Thanks, comment is updated